File: Łatwe/C++/Warunek w tablicy/WarunekWTablicy.cpp
#include <iostream>

int main() {
    int n;
    std::cin >> n;
    int numbers[n];
    for (int i = 0; i < n; i++)
        std::cin >> numbers[i];
    char sign;
    int x;
    std::cin >> sign >> x;
    for (int number : numbers) {
        if (sign == '<' && number < x)
            std::cout << number << std::endl;
        else if (sign == '>' && number > x)
            std::cout << number << std::endl;
    }
}

File: Łatwe/Java/Warunek w tablicy/WarunekWTablicy.java
import java.util.Scanner;

class WarunekWTablicy {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] numbers = new int[n];
        for (int i = 0; i < n; i++)
            numbers[i] = in.nextInt();
        char sign = in.next().charAt(0);
        int x = in.nextInt();
        for (int number : numbers) {
            if (sign == '<' && number < x)
                System.out.println(number);
            else if (sign == '>' && number > x)
                System.out.println(number);
        }
    }
}
